File: docs/components_page/components/pagination/navigation.R
library(dashBootstrapComponents)

pagination <- div(
  list(
    dbcPagination(max_value = 5, first_last = TRUE),
    dbcPagination(max_value = 5, previous_next = TRUE),
    dbcPagination(max_value = 5, first_last = TRUE, previous_next = TRUE)
  )
)

File: docs/components_page/components/pagination/navigation.jl
using DashBootstrapComponents, DashHtmlComponents

pagination = html_div([
    dbc_pagination(max_value = 5, first_last = true),
    dbc_pagination(max_value = 5, previous_next = true),
    dbc_pagination(max_value = 5, first_last = true, previous_next = true),
])
